### ISSUE TITLE
White color on pull request comment box

### ISSUE THREAD

**Issue by ItsDrike** [StylishThemes/GitHub-Dark#1258]
* **Browser**: Firefox 82.0.2 (64-bit)
* **Operating System**: Arch Linux (kernel: `linux 5.9.2.arch1-1`)
* **Link to page with the issue**: https://github.com/Codin-Nerds/Neutron-Bot/pull/11
* **Screenshot**:
![image](https://user-images.githubusercontent.com/20902250/97764064-baf3d980-1b0d-11eb-947f-62e8f15da361.png)

* **Description**: I'm not sure why but the comment box on any pull request on one of my repositories is white. I've checked other repositories and I couldn't replicate this behavior on other repositories.

**Comment by ItsDrike** [comment]
Actually I managed to find another occurence of this in one of my issues:
![image](https://user-images.githubusercontent.com/20902250/97764509-2a1dfd80-1b0f-11eb-9f2d-54c67de0cf6a.png)
link: https://github.com/microsoft/vscode-cpptools/issues/6388

**Comment by The-EDev** [comment]
the issue seems to be that `timeline-comment.current-user .timeline-comment-header` has no `background-color` property, so it uses the default one (`#9daccc`)

**Comment by The-EDev** [comment]
I added these to the stylesheet to fix the issue:
(I only added `background-color` for this one)
```
  .timeline-comment.current-user .timeline-comment-header {
    border-bottom-color: var(--color-auto-blue-2, #246);
      background-color: #1b2937;
  }
```
```
  .timeline-comment--caret.current-user::after {
    border-right-color: #1b2937;
  }
```

**Comment by silverwind** [comment]
Hmm, not seeing this, my comments are blue to myself as usual.

**Comment by ItsDrike** [comment]
> Hmm, not seeing this, my comments are blue to myself as usual.

Interesting, I even deleted the style and reinstalled it but it didn't help, maybe you're on an outdated version?

**Comment by silverwind** [comment]
No, latest master of course. Are you opted in to any GitHub experiments maybe?

**Comment by ItsDrike** [comment]
Not that I'm aware of, when i click on `Feature preview` on my profile there's nothing
![image](https://user-images.githubusercontent.com/20902250/97778963-422b6680-1b7b-11eb-9eaf-1012106c6ad5.png)

Not sure if this is what you meant though.

**Comment by silverwind** [comment]
Check if v4.2.41 just released solves it.

**Comment by The-EDev** [comment]
@silverwind seems to be fixed for me (after updating to 4.2.41)
